include/modules/clustering/simple_clustering_method.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "structures/cluster.hpp"   // for class Cluster
+
+/*! \brief This method compares junctions, and if they are equal, one will be removed from the junction list.
+ *
+ * \param junctions A vector of junctions
+ */
```

Comment:
I would rephrase a bit:

`This method clusters junctions by merging identical junctions into a single cluster object.`